### PR TITLE
[update] Refactor/api

### PR DESCRIPTION
**Pull request by Hamdrive**
## Overview

Updated the implementation of Twitter API to fetch the tweet creation date, and with the use of React-Moment it has been formatted to look identical to Twitter Embeds
In addition, some UI changes were also done

### CODE REVIEW

**Review by shivampandey0 — APPROVED** [review]
Clean implementation to fetch only the required info from API.

Good to merge

File: src/components/tweetCard/TweetCard.jsx [hunk 1]
```diff
@@ -1,29 +1,39 @@
...
 export const TweetCard = ({ tweet }) => {
   const {
     name,
     username,
     profile_image_url,
-    recentTweet: { id, text },
+    recentTweet: { id, text, created_at },
   } = tweet;
 
+  console.log(created_at)
```
Comment (prachi-sahani) [inline src/components/tweetCard/TweetCard.jsx]: can be removed

**Review by prachi-sahani — COMMENTED** [review]
All the changes are fine
Good implementation of "offline" page
Good to merge

File: src/components/tweetCard/TweetCard.jsx (hunk 1 above)
Comment (Hamdrive) [reply src/components/tweetCard/TweetCard.jsx]: Done

**Pull request merged**